docs/_includes/variation-content.html

```diff
@@ -59,10 +59,10 @@ <h1 class="component-header u-mb30">
 
     {% if page.variation_groups and page.variation_groups.size > 0 %}
     {% for variation_group in page.variation_groups %}
-    <section class="o-variation_group" id="{{ variation_group.variation_group_name | slugify }}">
+    <section class="o-variation-group" id="{{ variation_group.variation_group_name | slugify }}">
 
         {% if variation_group.variation_group_name and variation_group.variation_group_name != '' %}
-        <h2 class="o-variation_group_title">
+        <h2 class="o-variation-group-title">
```

Comment:
So in BEM this would actually be `o-variation-group_title` where `o-variation-group` is the Block (B) and `_title` is the Element (E) of that block.

Reply:
Ahh, I was wondering about this...
Updated, thanks!